harmoni_core/harmoni_decision/scripts/harmoni_decision/multiple_choice_decision.py

```diff
@@ -0,0 +1,88 @@
+            queue_size=1,
+        )
+        self.setup_scene()
+        
+    def start_scene(self, index_scene):
+        self.populate_scene(index_scene)
+        dp = SequentialPattern(self.name, self.script)
+        dp.start()
+        
+
+    def setup_scene(self):
+        for i in range(1,16):
+            self.sequence_scenes.append( {"background_choice":["container_1", ""],"background_cont":["container_2", ""], "background":["img_bkg", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_"+str(i)+"_Sfondo.png"], "text": "Background image", "choice_1":["img_1", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_"+str(i)+"_Comp.png"], "choice_2":["img_2", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_"+str(i)+"_Distr.png"], "choice_3":["img_3", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_"+str(i)+"_Target.png"]})
+        self.sequence_scenes.append({"background_cont":["container_2", ""], "background":["img_bkg", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_17_Sfondo.png"], "text": "Background image"}) 
+        self.sequence_scenes.append({"background_cont":["container_2", ""], "background":["img_bkg", "http://i3lab.elet.polimi.it/letssayresources/img/1_1_18_Sfondo.png"], "text": "Background image"})
+        return
+
+    def populate_scene(self, index_scene):
+        self.script[0]["steps"][0]["harmoni_web_default"]["trigger"] = "[{'component_id':'"+self.sequence_scenes[index_scene]["background"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["background"][1]+"'}, {'component_id':'"+self.sequence_scenes[index_scene]["background_cont"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["background_cont"][1]+"'}]"
+        self.script[0]["steps"][1]["harmoni_tts_default"]["trigger"] = self.sequence_scenes[index_scene]["text"]
+        if index_scene < self.choice_index:
+            self.script[0]["steps"][3]["harmoni_web_default"]["trigger"] = "[{'component_id':'"+self.sequence_scenes[index_scene]["choice_1"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["choice_1"][1]+"'},{'component_id':'"+self.sequence_scenes[index_scene]["choice_2"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["choice_2"][1]+"'},{'component_id':'"+self.sequence_scenes[index_scene]["choice_3"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["choice_3"][1]+"'}, {'component_id':'"+self.sequence_scenes[index_scene]["background_choice"][0]+"', 'set_content':'"+self.sequence_scenes[index_scene]["background_choice"][1]+"'}]"
+        with open(self.pattern_script_path, "w") as json_file:
+            json.dump(self.script, json_file)
+
+
+    def _event_click_callback(self, data):
+        "Received data"
+        print(data)
+        if "Target" in data.data: #IF CORRECT INCREMENT THE INDEX
+            self.index+=1
+        if self.index<self.max_index:
+            self.start_scene(self.index)
+
+
+
+
+
+if __name__ == "__main__":
+    try:
+        rospy.init_node("decision")
```

Comment:
General comment: we probably want to put all harmoni specific nodes in the "harmoni namespace", e.g. "harmoni/decision".